content/en/docs/contribute/style/diagram-guide.md

```diff
@@ -0,0 +1,447 @@
+---
+title: Diagram Guide
+linktitle: Diagram guide
+content_type: concept
+weight: 15
+---
+
+<!--Overview-->
+
+This guide shows you how to create, edit and share diagrams using the Mermaid javascript library. Mermaid.js allows you to generate diagrams using a simple markdown-like syntax inside markdown files. You can also use Mermaid to generate `.svg` or `.png` image files that you can add to your documentation.
+
+The target audience for this guide is anybody wishing to learn about Mermaid and/or how to create and add figures to Kubernetes documentation.
+
+The following diagram outlines the topics covered in this section. 
+
+{{< mermaid >}}
+flowchart LR
+subgraph m[Mermaid.js]
+direction TB
+S[ ]-.-
+C[build<br>diagrams<br>with markdown] -->
+D[on-line<br>live editor]
+end
+A[Why are diagrams<br>useful?] --> m
+m --> N[3 x methods<br>for creating<br>diagrams]
+N --> T[Diagram<br>styling]
+T --> X[Examples and<br>tips]
+
+    classDef box fill:#fff,stroke:#000,stroke-width:1px,color:#000;
+    classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+    class A,C,D,N,X,m,T box
+    class S spacewhite
+{{< /mermaid >}}
+<figure alt="diagram guide topics covered" style="padding-left: 310px; font-weight: bold">
```

Comment:
Is there a way we can use a CSS class rather than inline styles?
(could be a follow-up PR)

Reply:
Prefer follow-up PR.